client/src/auth/LogedoutRoute.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from "react";
+import { Route, withRouter } from "react-router-dom";
+import axios from 'axios';
+
+class LogedoutRoute extends Component {
+  initialMount = false;
+
+  // axios configuration
+  axiosSource = axios.CancelToken.source();
+  axiosInstance = axios.create({
+    baseURL: '/api/v1/',
+    cancelToken: this.axiosSource.token
+  });
+
+  state = {
+    isAuth: true,
+  }
+
+  componentWillUnmount() {
+    this.axiosSource.cancel('cancel isAuthenticated request');
+  }
+
+  componentDidMount() {
+    this.axiosInstance
+    .get('/isAuthenticated')
+      .then(() => {
+        this.isAuthorized();
+      })
+      .catch(() => {
+        this.initialMount = true;
+        this.setState({ isAuth: false });
+      });
+  }
+
+  componentDidUpdate(prevProps) {
```

Comment:
@ashatat And my comment in PrivateRoute.componentDidUpdate is relevant here too :)

Reply:
I think you're right. We don't need componentDidUpdate in LogedoutRoute component